direct_message/tests.py

```diff
@@ -1,3 +1,82 @@
-# from django.test import TestCase
+from . import models
+import pytest
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your tests here.
+
+CREATION_DATE = timezone.now()
+
+
+@pytest.fixture
+def new_user_sender():
+    return User(username='TestSender', first_name='Nick',
+                last_name='Birch', password='1234', email='Nick@email.com')
+
+
+@pytest.fixture
+def new_user_reciever():
+    return User(username='TestReciever', first_name='Andrew',
+                last_name='Glouberman', password='abcd', email='Andrew@email.com')
+
+
+@pytest.fixture
+def message0(new_user_sender, new_user_reciever):
+    return models.Message(sender=new_user_sender,
+                          reciever=new_user_reciever,
+                          creation_date=CREATION_DATE,
+                          message_text='test')
+
+
+def save_a_message(message0):
```

Comment:
I think you should add  `@pytest.mark.django_db()` annotation above this function.

Reply:
The save_a_message function is being called inside other functions that use `@pytest.mark.django_db()`